src/jshint.js

```diff
@@ -4242,7 +4242,18 @@ var JSHINT = (function() {
       // ImportClause :: ImportedDefaultBinding
       this.name = identifier();
       addlabel(this.name, { type: "unused", token: state.tokens.curr });
-    } else if (state.tokens.next.id === "*") {
+      if (state.tokens.next.value === ",") {
+        // ImportClause :: ImportedDefaultBinding , NameSpaceImport
```

Comment:
This is an oddball, but totally legit case. Coincidentally, @arv and I were talking about this grammar about two weeks ago (I had noticed that Traceur doesn't yet support it).